test/integration/Region.e2e.test.js

```diff
@@ -11,31 +11,37 @@ describe('Regions', () => {
         // Wait for the empty region layer to be present
         cy.getByTestId('ba-Layer--region');
 
-        // Assert that the region creator does not exist and no annotations are present
+        // Assert that the region creator does not exist
         cy.getByTestId('ba-RegionCreator').should('not.exist');
-        cy.get('[data-testid^="ba-AnnotationTarget"]').should('not.exist');
```

Comment:
I think we should leave this assertion in and make the selector more specific so that it only targets regions.